test/test_phabricatortest.yaml

```diff
@@ -0,0 +1,5 @@
+token: api-ptwgl2oswannngnw7aah74wk54cs
```

Comment:
Uh oh.  Should this token be public?

Reply:
It is a fake token, I will change it to 'xxx...'

Reply:
Ack, ok.  :)